_api/hasura/cron/syncCoSouls.ts

```diff
@@ -207,24 +217,118 @@ const syncCoSoulToken = async (
   );
 };
 
-async function updateCoSoulOnChain(cosoul: CoSoul, totalPGIVE: number) {
+const syncBatchCoSoulToken = async (
+  updatedCosouls: { cosoul: CoSoul; localPGIVE: number }[]
```

Comment:
`updatedCosouls` are the ones we want to update, not the ones that have been updated right? Maybe make variable name no longer the past tense. @sshmm